### DISCUSSION TITLE
are round_id's required to be date format?

### DISCUSSION THREAD

**Discussion by nickreich** [hubverse-org/.github#10] [Q&A]
This came up as @bsweger and I were thinking about the config files for a new hub. Do we assume that all round_id values will be dates, and in a parseable YYYY-MM-DD format? If so, that seems maybe usefully opinionated but also maybe a bit too restrictive? Anyways, just trying to recall what our motivation was for this decision, or how active of a decision it was. Any recollections or pointers would be useful. Maybe @annakrystalli @elray1 @LucieContamin have ideas?

**Comment by bsweger** [comment]
Adding some context:

This is the round_id definition in our schema: https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v2.0.1/tasks-schema.json#L27

Which contains examples like:

```
"examples": [
              "round-1",
              "2022-11-05",
              "origin_date"
                  ],
```

But there is code in `hubValidations` that is parsing filenames and expects the round_id piece to match the `YYYY-MM-DD` format: https://github.com/Infectious-Disease-Modeling-Hubs/hubValidations/blob/main/R/parse_file_name.R#L27

**Reply by LucieContamin** [reply]
If I remember correctly,  you can use a specific date, name or use a column name that contains the expected value, for example `origin_date`. But I guess the most frequent usage is a date format.

**Comment by elray1** [comment]
no, we do not have such a requirement. 

Edit: actually read Becky's comment, and saw that she linked to the schema already.  I'd say we should file this as an issue to clean up in hubValidations.

**Comment by annakrystalli** [comment]
Just dropping in link to relevant conversation regarding this topic during `hubValidations` development: https://github.com/Infectious-Disease-Modeling-Hubs/hubValidations/discussions/13
